### ISSUE TITLE
shadps4 is crashing whenever I try to install a game

### ISSUE THREAD

**Issue by nycolas281** [shadps4-emu/shadPS4#692]
When I try to install a pkg, the emulator closes and when I open it, only part of the game appears, for example, Bloodborne has 31 GB but it appears that it has 11 MB

**Comment by powerpuffboysz** [comment]
What version are you using? What os? I ask because i had a similar issue trying to install the uncharted nathan drake collection on linux with the appimage that seemingly fixed itself after an os update

**Comment by Andistive** [comment]
Having the exact same issue installing inFAMOUS second son. I am using 0.2.0ql with Windows 10.

**Comment by rmueller83** [comment]
Related to #726 ?
Look into the Windows Even Viewer under "Windows Logs" / "Application".

**Comment by KubokPekla** [comment]
I have the same issue on Windows 10, I've tried the two latest builds in Windows-QT, 2024-09-05-09107cc and 2024-09-05-00ed72e and they both have the same issue, that is it crashes to desktop almost immediately without an error message. This is with Bloodborne, though I'll try PT in a bit to see if maybe there's something wrong with the PKG file. I can install the 1.09 patch for some reason and it recognizes that, but otherwise it just leaves me with an 11.6MB install that obviously crashes the emulator when I try to run it.

Windows Event Viewer doesn't really help, it just says it can't access the file for a multitude of possible different reasons, most of which I've been able to determine aren't the problem with CHKDSK, trying to install the PKG from a different drive, running ShadPS4 on another drive entirely, seeing that my hard drive is in fact connected and being able to access other files on the disk without issue. The only thing that it mentions that I'm not entirely sure about is it possibly having something to do with storage drivers, but I've never had to mess with that and I'm having a hard time believing that's the cause, no clue though tbh.

I wasn't exactly expecting smooth sailing but damn, can't even install the game lol.
